todo-app/src/Main.js

```diff
@@ -1,24 +1,52 @@
 import React from 'react';
-import ListItem from "./List-Item";
 import todoItems from "./todos.json";
-import AddTodo from "./Add-Todo";
+import ListItem from "./List-Item";
+import TodoInput from "./Todo-Input";
+
 
+const lastId = todoItems[todoItems.length - 1].id;
```

Comment:
This is not a safe way to check for IDs. You should use an array method to produce an array of all current IDs, and then use something like `Math.max` to determine the highest value.

If I create four todos from scratch, I'll have todos with IDs of `1, 2, 3, 4`. Now imagine I delete #3, and now have `1, 2, 4`. For the next todo created, your system here would attempt to create another todo with an ID of 4.